### ISSUE TITLE
How did you split the dataset?

### ISSUE THREAD

**Issue by Kihensarn** [orpatashnik/StyleCLIP#86]
It's a wonderful work to manipulate face by using only text. 
But when I try the method in mapper folder，I see that the number of training samples is 24176 and the number of test samples is 2824 in the train_face.pt and test_face.pt, respectively. I am confused that the total numbers is not 30000 because there are 30000 samples in CelebA-HQ dataset. And could you also let me know the corresponding filename in the train_face.pt and test_face.pt？

**Comment by orpatashnik** [comment]
Hi @Kihensarn ,

The training set is FFHQ, and the test set is the CelebA-HQ test set.
Unfortunately, I don't have the mapping between the indices and the filenames but it should be the normal order of each set.
Why do you need the correspondence?

**Comment by Kihensarn** [comment]
Thanks for your reply!

But I am still confused about the number of training set because there are 70000 samples in the FFHQ while there are 24176 samples in the train_face.pt. Is it a subset of the FFHQ dataset?

And the reason why I want to know the correspondence is that I want to split the dataset you provided according to the attribute annotation file provided by CelebA.

**Comment by ssxxx1a** [comment]
confused also....

**Comment by zhumengzj** [comment]
这是来自QQ邮箱的假期自动回复邮件。你好，来信已收到，谢谢！
